Resources/Prototypes/_NF/Entities/Materials/materials.yml

```
@@ -10,3 +10,23 @@
   - type: Tag
     tags:
     - MaterialHideRosyMothroach
+
+- type: entity
+  parent: MaterialBiomass
+  id: MaterialBiomass50
+  suffix: Fifty
```

```suggestion
  suffix: 50
```

Also you should make it a no spawn entity probably

---

1. Categories don't inherit (afaik)
2. Probably no harm in keeping it around, especially if this is to be mapped in service shuttles
3. Alternatively to 2, consider a Shuttle suffixed biogenerator preloaded with biomass.